control-plane/agents/src/bin/core/volume/specs.rs

```diff
@@ -1111,15 +1111,6 @@ pub(crate) async fn get_volume_target_node(
     status: &VolumeState,
     request: &PublishVolume,
 ) -> Result<NodeId, SvcError> {
-    // We can't configure a new target_node if the volume is currently published
```

Comment:
we could keep this and check if request is not a republish?

Reply:
Yeah. Right